### DISCUSSION TITLE
deploy uBOL in Complete Filtering Mode

### DISCUSSION THREAD

**Discussion by matchstickboy** [uBlockOrigin/uBOL-home#135] [Q&A] [answered]
Hi All,

Apologies if this is addressed elsewhere and I just missed it.  but I'm curious whether or not it would be possible to deploy uBOL with the  Default Filtering Mode set to Complete when the extension is installed.

Currently testing automatic deployment to Edge and Chrome via Active Directory GPO, so maybe setting the filtering mode using this deployment method is not possible.  but would be very interested to know if there is a way to accomplish this.

Thanks,

Chris

**Comment by gorhill** [comment, marked as answer]
Not possible, see <https://github.com/uBlockOrigin/uBOL-home/issues/113>

Until Google Chrome provides a way to admin-declare optional host permissions for a given extension, this is not possible, as currently the browser requires a user gesture for the dialog to allow/deny optional host permissions.

**Reply by matchstickboy** [reply]
Thought as much.  Thank you for clarifying.  Apologies for not seeing the earlier discussion where this was addressed.

Thanks for all of your work!
